src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -139,6 +141,11 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, {}> {
                         {this.getDropdownOptionAppend(opt)}
                     </li>
                 );
+
+                return opt.disabled && opt.disabledTooltip
+                    ? <Tooltip {...opt.disabledTooltip}>{formattedOption}</Tooltip>

```

Comment:
This will wrap the `li` element in a span, I'd prefer to put the span inside the li if possible

Reply:
this doesn't seem to affect a list per se:  

<img width="1190" alt="screen shot 2018-05-28 at 10 04 17 am" src="https://user-images.githubusercontent.com/9539763/40618160-a531fa1a-625e-11e8-96d7-13c31fc8bb85.png">  
  
I tried your suggestion but it breaks the style :S   
  
is it a must do for you ?